examples/vitis_ai/retinaface_ptq_cpu/user_script.py

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import os
+import numpy as np
+import argparse
+from PIL import Image
+import onnxruntime
+import cv2
+from onnxruntime.quantization.calibrate import CalibrationDataReader
+from olive.constants import Framework
+from olive.evaluator.accuracy import AccuracyScore
+from olive.model import OliveModel
+
+
+class ResNetDataReader(CalibrationDataReader):
+
+    def __init__(self, calibration_image_folder, augmented_model_path):
+        self.image_folder = calibration_image_folder
+        self.augmented_model_path = augmented_model_path
```

Comment:
How about to accept model path in under the component of vitis_ai?
In some cases, vitis quantization could be after conversion where the model path is generated dynamically. For examples:
1. conversion -> optimization -> vistis quantization -> ....
In this optimization chain, user may not know the model path, then it is hard to set it in olive config.